messages/maintainer_access.md

```diff
@@ -0,0 +1,19 @@
+Hello {NAME},
+
+You have been nominated as {role} and the Coordinating Committee enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}
+
+You can see the formal description of the maintainers role at http://www.astropy.org/team.html. If you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project, or if absolutely necessary, the Coordination Committee.
```

Comment:
Since this is the first communication with the person nominated to be a maintainer and they are expected to answer whether they accept the nomination, would it make sense to elaborate a little more about the expectations from a maintainer? I know that this might be hard to frame since it is a "voluntary" task, but it might help make the decision. I checked http://www.astropy.org/team.html -- it just lists the tasks a maintainer is supposed to do without any concrete ideas about time commitment. This is the first thing that came to my mind when I thought about what I would wonder about if I received such an email/correspondence, but I don't know if there is a specific answer to this!